boms/cloud-oss-bom/pom.xml

```diff
@@ -47,21 +47,21 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <guava.version>31.0.1-jre</guava.version>
+    <guava.version>31.1-jre</guava.version>
```

Comment:
are we missing a number here?

Reply:
Rest assured. Guava's release has sometimes 2 numbers and other times 3 numbers.

https://search.maven.org/artifact/com.google.guava/guava

(Also we have a check to ensure the artifacts in the BOM exists in Maven Central. It would catch any typo in version or artifact IDs)